client/src/components/ApplicantProfile.js

```diff
@@ -57,21 +57,22 @@ class ApplicantProfile extends React.Component {
       applicationStatus: "",
       rightToWork: null,
       email: "",
-      skills: []
+      skills: [],
+      about: ""
     };
   }
 
   componentDidMount() {
-    const user_id = 2; //This is hard coded because we don't have user login information yet
-
+    const user_id = null;
```

Comment:
why did you cahnge this from 2 to null .. does this work?

Reply:
it didnt give me error. Also, i am working on it in another branch. User id will be taken form localStorage.

Reply:
it didn't give error but it won't get anything ... leave it as 2 then